WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingCoordinator.swift

```diff
@@ -167,6 +169,29 @@ private extension StoreOnboardingCoordinator {
                             subtitle: Localization.StoreNameNotice.subtitle)
         noticePresenter.enqueue(notice: notice)
     }
+
+    func showWooPaySetupCelebrationView() {
+        wooPaySetupCelebrationViewBottomSheetPresenter = buildBottomSheetPresenter()
+        let controller = WooPaymentSetupCelebrationHostingController(onTappingDone: { [weak self] in
+            self?.wooPaySetupCelebrationViewBottomSheetPresenter?.dismiss()
+            self?.wooPaySetupCelebrationViewBottomSheetPresenter = nil
+        })
+        wooPaySetupCelebrationViewBottomSheetPresenter?.present(controller, from: navigationController)
+    }
+}
+
+// MARK: Bottom sheet helpers
+//
+private extension StoreOnboardingCoordinator {
+    func buildBottomSheetPresenter() -> BottomSheetPresenter {
+        BottomSheetPresenter(configure: { bottomSheet in
+            var sheet = bottomSheet
+            sheet.prefersEdgeAttachedInCompactHeight = true
+            sheet.largestUndimmedDetentIdentifier = .none
+            sheet.prefersGrabberVisible = true
+            sheet.detents = [.medium(), .large()]
```

Comment:
Nit: I think we don't need the large detent since the content should fit the medium one right?

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/83c0f039-14dc-4bb8-a40a-b0aa8c06fd25" width=320 />

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10594/commits/f089d27cbeea23679e8d08708bae5ad18e9d0ccc